client/style.css

```diff
@@ -138,6 +138,221 @@ h1{
   border-color: blue;
 }
 
+.title-bar {
+  background-color: grey;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  color: white;
+  font-size: 1.6rem;
+  font-style: italic;
+  text-decoration: underline;
+  padding: 5px;
+}
+
+.logo {
+  font-weight: 600;
+}
+
+.sitewide-announce {
+  text-align: center;
+  font-size: 0.8rem;
+  margin: 2px;
+}
+
+.overview-column-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-bottom: 2%;
+}
+
+.image-gallery-container {
+  position: relative;
+  max-width: 60%;
+  min-width: 60%;
+  padding-top: 45%;
+}
+
+.image-gallery {
+  background-color: lightgray;
+  position: absolute;
+  width: 100%;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+.image-gallery-fullscreen-toggle {
+  position: absolute;
+  float: left;
```

Comment:
I think on absolute positioned elements, you can use `left: 0` as float won't affect elements that aren't in the normal flow of the document? Not 100% on this

Reply:
The float banished a pesky margin that refused to zero out the normal way. It's a total hack, to say it explicitly.

Reply:
(And one that unless I find another workaround to get rid of a big empty margin, needs to stay in.)